docs/src/main/sphinx/release/release-368.md

```diff
@@ -0,0 +1,51 @@
+# Release 368 (xx Jan 2022)
+
+## General
+
+* Improve performance of arithmetic operations involving decimals with precision
+  larger than 18. ({issue}`10051`)
+* Introduce `prioritize-utilization` query execution policy that aims to improve
+  resource utilization for queries that involve joins and for concurrent
+  workloads. ({issue}`10350`)
+* Fix delegated refresh materialized views queries not transitioning into
+  `RUNNING` state. ({issue}`10360`)
+* Allow setting per task memory limits via `query.max-total-memory-per-task`
+  config property or via `query_max_total_memory_per_task` session property.
+  ({issue}`10308`)
+
+## Hive connector
+
+* Improve performance of decoding decimal values with precision larger than 18
+  in ORC, Parquet and RCFile data. ({issue}`10051`)
+* Disallow querying the properties system table for Delta Lake tables.
```

Comment:
Maybe add a qualifier indicating that Delta Lake tables are unsupported, and that's why querying their properties tables is being disallowed. Otherwise, it raises the question of why we're removing functionality.

Reply:
makese sense.
 
```
* Disallow querying the properties system table corresponding to Delta Lake tables, since Delta Lake tables are not supported. This fixes the previous behavior of silently returning incorrect values. ({issue}`10447`)
```

Reply:
Updated